src/formatDate.js

```diff
@@ -51,6 +51,49 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dateArr = date.split(fromFormat[3]);
+  const mounth = dateArr[fromFormat.indexOf('MM')];
+  const day = dateArr[fromFormat.indexOf('DD')];
+  let year = '';
+
+  if (fromFormat.indexOf('YY') === -1) {
+    year = dateArr[fromFormat.indexOf('YYYY')];
+  } else {
+    year = dateArr[fromFormat.indexOf('YY')];
+  }
```

Comment:
you can create date object with format keys and values. And use a loop to set it:

{
  'YYYY': 2020,
  'MM': 02,
  ...
}